app/services/location_data_service.rb

```diff
@@ -25,7 +25,7 @@ def populate_holding_locations
       library_record = Locations::Library.find_by(code: library.code)
       holding_locations(library.code).each do |holding_location|
         Locations::HoldingLocation.new do |location_record|
-          location_record.label = "#{library_record.label} - #{holding_location.name}"
+          location_record.label = (library_record.label != holding_location.name ? "#{library_record.label} - #{holding_location.name}" : library_record.label)
```

Comment:
Can we switch the order of the ternary around so that it compares on equality like `library_record.label == holding_location.name`